<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialVisualizationWidget</class>
 <widget class="QWidget" name="MaterialVisualizationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>Name of component</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
     <item>
      <widget class="QComboBox" name="dataArrayNameComboBox">
       <property name="toolTip">
        <string>Name of data array</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="componentNameComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Color space:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="colorSpaceComboBox">
       <property name="toolTip">
        <string>Color space</string>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <item>
        <property name="text">
         <string>RGB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HSV</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LAB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diverging</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="loadTFButton">
       <property name="toolTip">
        <string>Load transfer funtion from file</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/SolverSetup.qrc">
         <normaloff>:/contourModeling/icons/open.png</normaloff>:/contourModeling/icons/open.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="saveTFButton">
       <property name="toolTip">
        <string>Save transfer function to file</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/SolverSetup.qrc">
         <normaloff>:/contourModeling/icons/save.png</normaloff>:/contourModeling/icons/save.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QmitkColorTransferFunctionCanvas" name="colorTransferFunctionCanvas" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="toolTip">
      <string>Left-click to select a point or add a new point.
Hold left mouse button to move selected point.
Click right mouse button to delete a point.
Double-click left mouse button to change color of a point.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="valueLineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>x-coordinate of currently selected point</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_textLabelX">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="toolTip">
        <string>Edit x-coordinate (grayvalue) of currently selected point.</string>
       </property>
       <property name="text">
        <string>Value</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Unassigned:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkColorPickerButton" name="nanColorButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>The color for the faces where material value has not been assigned</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="resetTFButton">
       <property name="toolTip">
        <string>Reset transfer function to default</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkColorTransferFunctionCanvas</class>
   <extends>QWidget</extends>
   <header>QmitkColorTransferFunctionCanvas.h</header>
  </customwidget>
  <customwidget>
   <class>ctkColorPickerButton</class>
   <extends>QPushButton</extends>
   <header location="global">ctkColorPickerButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../uk.ac.kcl.VesselMeshing/resources/VesselMeshing.qrc"/>
  <include location="../../resources/SolverSetup.qrc"/>
 </resources>
 <connections/>
</ui>
